src/components/multi-line-input/MultiLineInput.tsx

```diff
@@ -25,35 +25,34 @@ export const MultiLineInput = ({
   ...rest
 }: MultiLineInputProps) => {
   const { t } = useTranslation();
-  const { register, watch, setValue } = useFormContext();
+  const { register, watch } = useFormContext();
 
   const value = watch(name, defaultValue);
-  const fields = Array.isArray(value) && value.length !== 0 ? value : [""];
+  const [indexes, setIndexes] = useState<number[]>(
```

Comment:
I think you can omit the type here as it should be inferrable.
```suggestion
  const [indexes, setIndexes] = useState(
```